webapp/javascript/util/formatDate.js

```diff
@@ -10,32 +11,65 @@ const multiplierMapping = {
   y: "year",
 };
 
-export default function humanReadableRange(until, from) {
-  if (until === "now" && typeof from === "string") {
-    const m = from.match(/^now-(?<number>\d+)(?<multiplier>\D+)$/);
-    if (m && multiplierMapping[m.groups.multiplier]) {
-      let multiplier = multiplierMapping[m.groups.multiplier];
-      if (m.groups.number > 1) {
-        multiplier += "s";
-      }
-      const readableDateForm = {
-        range: `Last ${m.groups.number} ${multiplier}`,
-        from: moment().add(-m.groups.number, multiplier).toDate(),
-        until: moment().toDate(),
```

Comment:
Would be nice to add comment about type/form of arguments and type/form of functions return. This will make it easier for someone to understand exactly what this functions does.